packages/patternfly-4/react-core/src/components/AboutModal/AboutModalContainer.js

```diff
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types';
 import AboutModalBoxContent from './AboutModalBoxContent';
 import AboutModalBoxHeader from './AboutModalBoxHeader';
 import AboutModalBoxHero from './AboutModalBoxHero';
-import AboutModalBoxLogo from './AboutModalBoxLogo';
```

Comment:
Why is this being removed as part of this upgrade?

Reply:
The box logo was removed entirely from pf-next v1.0.107. Since the styles are gone from pf-next, wasn't sure how to make the logo optional

Reply:
https://github.com/patternfly/patternfly-next/pull/1135 @tlabaj